lib/form/html.js

```diff
@@ -113,10 +113,14 @@ const addRequired = (html, input) => {
 const renderer = (type, errors) => {
   return new Promise((res) => {
     let rendered = type.attributes.map(input => {
-      const inputs = Object.keys(input.inputs).length;
+      let inputs = Object.keys(input.inputs).length;
+      if (Array.isArray(input.inputs) && input.inputs.length > 0) {
```

Comment:
Can you explain to me what you're doing here? Why are hard-coding the first item to grab?